CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+5.0.2 / 2020-12-29
```

Comment:
Seems you did not use the newest master as a base. 5.1.0 is the newest version, so this should either be 5.1.1 or 5.2.0. As it adds functionality it should be 5.2.0

Reply:
Hmm yeah I think I used newest master but package.json may have been out of date?

Reply:
https://github.com/Twipped/joi-to-swagger/blob/master/package.json#L3

Reply:
Ah I forgot to push changes of 5.1.0, sorry.